### PR TITLE
Patient details and comments

### PR DESCRIPTION
**Pull request by jHelsing**
The following features have been added:

- The detailed view of a patient has been redesigned
- It is possible to add treatments, diagnosis and notes to a patient which will then be shown in the history.

Features still missing that require more functionality to be implemented in other parts of the app:

- Showing who added what comment, i.e. which doctor diagnosed the patient with X.
- Adding photos to diagnosis and notes. (This is partly implemented but chosing images does not work on my computer and it is therefore not included in the PR)
- Changing permissions for adding different comments, i.e. a nurse can only add a note etc.

### CODE REVIEW

File: src/providers/comment/comment.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import { Injectable } from '@angular/core';
+import { AngularFirestore, AngularFirestoreCollection } from 'angularfire2/firestore';
+import { Comment } from '../../models/comment';
+import { Observable } from 'rxjs/Observable';
+import * as firebase from 'firebase';
+
+@Injectable()
+export class CommentProvider {
+  private commentsCollection: AngularFirestoreCollection<Comment>;
+  private comments: Observable<Comment[]>;
```
Comment (ckarlss0n) [inline src/providers/comment/comment.ts]: The value of `comments` is never read. Should it be removed?

File: src/pages/patient-detail/patient-detail.html [hunk 2]
```diff
@@ -1,24 +1,60 @@
...
-  </ion-item>
-  <ion-item>
-    <button ion-button (click)="goToUpdatePatient(patient)">Update</button>
-  </ion-item>
+  <ion-list>
+    <ion-card>
+      <ion-card-header>Basic Info</ion-card-header>
+      <ion-card-content>
+        <p><strong>Name: </strong> {{patient.name}}</p>
+        <p><strong>Date of Birth: </strong>{{patient.age}}</p>
```
Comment (davidemdot) [inline src/pages/patient-detail/patient-detail.html:21]: We should replace the age by the date of birth in the DB, if we finally going to choose that way.

**Review by davidemdot — APPROVED** [review]

File: src/providers/comment/comment.ts (hunk 1 above)
Comment (oskarlignell) [reply src/providers/comment/comment.ts]: I think it should be removed. The patient-detail.ts is where the observable is used for comments on a patient view, not necessary to have it here as well (?)

File: src/components/add-comment/add-comment.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
+<!-- Generated template for the AddCommentComponent component -->
```
Comment (oskarlignell) [inline src/components/add-comment/add-comment.html]: No big deal but could be removed

**Review by ckarlss0n — APPROVED** [review]

**Pull request merged**